sdk/test/Performance/CborPerformanceBenchmarks/CborPerformanceBenchmarksRunner/EchoBenchmarks/AllTypesBenchmarks.cs

```diff
@@ -0,0 +1,134 @@
+﻿using Amazon;
+using Amazon.Echo;
+using Amazon.Echo.Model;
+using Amazon.Echo.Model.Internal.MarshallTransformations;
+using Amazon.Runtime;
+using Amazon.Util;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Order;
+using System;
+using System.Text;
+
+namespace CborPerformanceBenchmarksRunner.EchoBenchmarks;
+
+public class AllTypesBenchmarks : BaseBenchmarks
+{
+    protected Random _random = new Random();
+
+    [Params(0)]
+    public override int DimensionValue { get; set; }
+    public override string Service { get; } = "EchoService";
+    public override string TestCase { get; } = "All types";
+
+    protected IAmazonEcho _client;
+    public override void CreateServiceClient()
+    {
+        _client = new AmazonEchoClient(new AmazonEchoConfig
+        {
+            DisableRequestCompression = true,
```

Comment:
Curious, does this matter for the benchmark? I thought the model had to explicitly enable it for the SDK to compress the request...